app/code/Magento/InventoryExportStockApi/Test/Api/ExportStockIndexDataTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryExportStockApi\Test\Api;
+
+use Magento\TestFramework\TestCase\WebapiAbstract;
+use Magento\Framework\Webapi\Rest\Request;
+
+class ExportStockIndexDataTest extends WebapiAbstract
+{
+    const API_PATH = '/V1/inventory/dump-stock-index-data';
+    const SERVICE_NAME = 'inventoryExportStockApiExportStockIndexDataV1';
+
+    const EXPORT_PRODUCT_COUNT = 1206;
```

Comment:
1206 seems to be a magical number used during testing. Should we adjust it to 6 or even calculate dynamically?